app/views/home/index.html.erb

```diff
@@ -6,27 +6,18 @@
     </div>
   </div>
   <!-- "Welcome" section -->
-  <section class="homepage-welcome-banner dm-gradient-banner-318" role="region" aria-label="Homepage welcome banner">
-    <div class="grid-container height-full">
-      <div class="grid-row height-full flex-align-center">
+  <section class="homepage-welcome-banner usa-section padding-top-3 padding-bottom-7" role="region" aria-label="Homepage welcome banner">
+    <div class="grid-container">
+      <div class="grid-row">
```

Comment:
```suggestion
      <div class="grid-row text-center text-white flex-justify-center">
```
1. Set the `text` properties up here so you don't have to specify per element
2. use USWDS flexbox properties and you can remove the `grid-col-offset-1` and other margin properties on the `<p>` tag. As much as possible, we want USWDS and the automagic of flexbox to do the heavy lifting on calculating margins for us.